src/test/kotlin/no/nav/helsearbeidsgiver/utils/json/SerializationUtilsKtTest.kt

```diff
@@ -155,6 +158,22 @@ class SerializationUtilsKtTest :
 
                 json shouldBe "\"$uuid\""
             }
+
+            test("serialiserer korrekt fra Orgnr til JsonElement") {
+                val orgnr = Orgnr.genererGyldig()
+
+                val json = orgnr.toJson().toString()
+
+                json shouldBe "\"$orgnr\""
```

Comment:
Jeg antar at toString for Orgnr gir det på dette formatet "1234556"?

Reply:
Stemmer.